### PR TITLE
docs: update readmes for api2teams

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/api2teams/README.md [hunk 1]
```diff
@@ -64,41 +40,26 @@ api2teams -h # show help message
...
-- [Adaptive card official site](https://adaptivecards.io/)
-- [Adaptive card designer](https://adaptivecards.io/designer)
+## Current limitations
+1. The `api2teams` doesn't support Open API schema version < 3.0.0
+1. The `api2teams` doesn't support Authorization property in Open API specification
+1. The `api2teams` doesn't support `webhooks` property and it would be ignored during convert
+1. The `api2teams` doesn't support `oneOf`, `anyOf`, `not`keyword (It only support `allOf` keyword currently)
+1. The generated Adaptive Card doesn't support array type. 
+1. The generated Adaptive Card doesn't support file upload.
+1. The generated Teams app can only contain up to 10 items in the command menu.
```
Comment (SLdragon) [inline packages/api2teams/README.md:58]: Missing: Only GET operations are supported.

File: packages/api2teams/src/resources/project-template/README.md [hunk 2]
```diff
@@ -35,53 +40,40 @@ The converted app is a normal command bot TeamsFx project that will contain foll
...
+- [Add notifications to the project](https://aka.ms/teamsfx-notification-new)
+
+## Current limitations
+1. The `api2teams` doesn't support Open API schema version < 3.0.0
+1. The `api2teams` doesn't support Authorization property in Open API specification
+1. The `api2teams` doesn't support `webhooks` property and it would be ignored during convert
+1. The `api2teams` doesn't support `oneOf`, `anyOf`, `not`keyword (It only support `allOf` keyword currently)
+1. The generated Adaptive Card doesn't support array type. 
+1. The generated Adaptive Card doesn't support file upload.
+1. The generated Teams app can only contain up to 10 items in the command menu.
```
Comment (SLdragon) [inline packages/api2teams/src/resources/project-template/README.md:79]: Missing : Only GET operations are supported.

File: packages/api2teams/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply packages/api2teams/README.md:59]: Updated

File: packages/api2teams/src/resources/project-template/README.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/api2teams/src/resources/project-template/README.md:80]: Updated

**Pull request merged**